Cargo.toml

```diff
@@ -1,9 +1,16 @@
 [package]
-name = "cloudevents-rust-sdk"
+name = "cloudevents"
```

Comment:
the name `cloudevents` is not available on docs.rs or creates.io, may we keep the old name?
Or we take `cloudevents-sdk`?

https://crates.io/crates/cloudevents
https://docs.rs/cloudevents/0.1.1/cloudevents/

Reply:
I think `cloudevents-sdk` is fine then, i need to reserve it now i suppose